code/renderers/react/src/entry-preview-docs.ts

```diff
@@ -1,29 +1,8 @@
-import type { ArgTypesExtractor } from 'storybook/internal/docs-tools';
-import { enhanceArgTypes, extractComponentDescription } from 'storybook/internal/docs-tools';
-import type { ArgTypesEnhancer, DecoratorFunction } from 'storybook/internal/types';
+import type { DecoratorFunction } from 'storybook/internal/types';
 
-import { extractArgTypes } from './docs/extractArgTypes';
 import { jsxDecorator } from './docs/jsxDecorator';
 import type { ReactRenderer } from './types';
 
-export const parameters: {
-  docs: {
-    story: {
-      inline: boolean;
-    };
-    extractArgTypes: ArgTypesExtractor;
-    extractComponentDescription: (component?: any) => string;
-  };
-} = {
-  docs: {
-    story: { inline: true },
-    extractArgTypes,
-    extractComponentDescription,
-  },
-};
```

Comment:
I removed the types here during the migration. It was pretty confusing to read the same code twice, and I don't think it serves any purpose to type the object exactly like it's actual properties, it just means you have to change the code in two places whenever you make changes.